### ISSUE TITLE
Feature Request - Fine tuning script/information

### ISSUE THREAD

**Issue by RelaxingVR** [DepthAnything/Depth-Anything-V2#124]
Is it possible to get information on how to fine-tune using custom datasets? I wanted to generate my own with different text overlays/HUDs/etc to try and get it to handle those better - but I have been struggling to see how to do it.

I saw the training script in the metric-depth folder and I know that is coded expecting KITTI or HyperSim but should that be possible to alter and just provide my own rgb + depth images and it would train on them?

If not the few areas I'd love to see improvements on are - Heads Up Displays/text overlays, glass (cars/windows), ropes/lines. I think I can generate datasets for HUDs/text overlays easily enough in Unity if anyone would like to collaborate on fine-tuning for that!

**Comment by LiheYoung** [comment]
Hi, I think this awesome blog may be helpful to you: https://huggingface.co/blog/Isayoften/monocular-depth-estimation-guide

**Comment by RelaxingVR** [comment]
Thank you! I tried googling and hadn't managed to find anything!
